snuba/cli/subscriptions_executor.py

```diff
@@ -116,10 +122,11 @@ def subscriptions_executor(
     stream_loader = storage.get_table_writer().get_stream_loader()
     result_topic_spec = stream_loader.get_subscription_result_topic_spec()
     assert result_topic_spec is not None
+    physical_topic_name = result_topic_spec.get_physical_topic_name(slice_id)
```

Comment:
Just to ensure we are on the same page. 
Will it be possible to have a different scheduled topic per slice but share the result topic? 
the fact the snuba part of the pipeline (ingestion and subscription) is sliced does not mean the alert feature in the product is (which consumes the result topic)

Reply:
That should be the default behavior right now. The slice boundary ends at writing the result to the subscription result topic. All results have to be written to an unsliced topic name. 

@ayirr7 - What this means is that when you are resolving topic names based on slices, you should only do that for source topic names. Not for the result topic names.

Reply:
Oh, right, good point. I didn't quite realize the slicing boundary ends for the results topics (they aren't sliced). Thanks for the clarifications.